srcpkgs/python-args/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'python-args'
+pkgname=python-args
+version=0.1.0
+revision=1
+wrksrc="${pkgname#*-}-$version"
+build_style=python-module
+pycompile_module="args.py"
+hostmakedepends="python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel"
+short_desc="Argument parsing for humans"
+maintainer="Emerson Ferreira <me@skrps.me>"
+license="BSD-2-Clause"
+homepage="https://github.com/kennethreitz/${pkgname#*-}"
+distfiles="$homepage/archive/v$version.tar.gz"
+checksum=ead9ed80cbb64179a768d2aba8edba9f83dc2e33dc91ac07a7479f3d74119568
+
+post_install() {
+	vlicense LICENSE
+}
+
+python3-args_package() {
```

Comment:
forgot to set up short_desc= , depends=

Reply:
args doesn't have dependencies, aside from python-nose for testing purposes

Reply:
Should I put `depends="python"` or something?

Reply:
depends="python3"

also short_desc should be

short_desc="${short_desc/Python2/Python 3}"

and you can add (Python 2) on the short_desc of the main package

Reply:
alright